File: net/merge/Merger_pdf_svg.cs
namespace aspose_snippets.net
{
    public static partial class Merger
    {
        public static void pdf_to_svg()
        {
            const string pathSource1 = "../../TestData/test.pdf";
            const string pathSource2 = "../../TestData/Second/test.pdf";

            //read pdf file to Aspose Document
            var firstDoc = new Aspose.Pdf.Document(pathSource1);
            var secondDoc = new Aspose.Pdf.Document(pathSource2);

            //create empty pdf document
            var outputDoc = new Aspose.Pdf.Document
            {
                //set less memory usage with unload instead of fast performance
                EnableObjectUnload = true
            };

            foreach (var page in firstDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            foreach (var page in secondDoc.Pages)
            {
                //add page from one document to another directly
                outputDoc.Pages.Add(page);
            }

            //save pdf to svg
            outputDoc.Save("test.svg",
                            new Aspose.Pdf.SvgSaveOptions
                            {
                                //scale the output document from typographic points to pixels
                                ScaleToPixels = true
                            });
        }
    }
}

File: python/merge/Merger_pdf_svg.py
from aspose.pdf import (
    SvgSaveOptions,
    Document
)


def pdf_to_svg(self):
    pathSource1 = "../../TestData/test.pdf"
    pathSource2 = "../../TestData/Second/test.pdf"

    # read pdf file to Aspose Document
    firstDoc = Document(pathSource1)
    secondDoc = Document(pathSource2)

    # create empty pdf document
    outputDoc = Document()

    # set less memory usage with unload instead of fast performance
    outputDoc.enable_object_unload = True

    for page in firstDoc.Pages:
        # add page from one document to another directly
        outputDoc.Pages.Add(page)

    for page in secondDoc.Pages:
        # add page from one document to another directly
        outputDoc.Pages.Add(page)

    opt1 = SvgSaveOptions()
    # scale the output document from typographic points to pixels
    opt1.scale_to_pixels = True

    # save pdf to svg
    outputDoc.Save("test.svg", opt1)
